js/modules/k6/http/request.go

```diff
@@ -104,8 +99,8 @@ func (h *HTTP) Request(ctx context.Context, method string, url goja.Value, args
 		params = args[1]
 	}
 
-	req, err := h.parseRequest(ctx, method, u, body, params)
-	if err != nil {
+	req, err := h.parseRequest(ctx, method, url, body, params)
+	if err != nil || req == nil {
 		return nil, err
```

Comment:
One difference here with how this was done in #1853 is that the returned `Response` will be `undefined` in JS.

To me this makes more sense since if the request failed during parsing and we never made the call, we shouldn't create a `Response` object for it either.

Reply:
No longer an issue after f2665be0.